packages/fxa-auth-server/lib/senders/templates/layouts/fxa.txt

```diff
@@ -0,0 +1,9 @@
+{{{body}}}
+
+Mozilla. 331 E Evelyn Ave, Mountain View, CA 94041
+
+{{t "Mozilla Privacy Policy" }}
+{{{privacyUrl}}}
```

Comment:
In the process of moving this to the layout, I also opted to introduce some newline characters before links. Most of our text emails were an impenetrable wall of text previously, it's like no-one on the team had ever really looked at them.

Just by putting URLs on their own line, it makes them a bit more readable without causing any string changes for localisers. But bear it in mind if you're reviewing the changes locally, if you diff the before and after rendered text emails, you'll see the newlines show up.